### PR TITLE
New Packages: libhangul, ibus-hangul

### PR DESCRIPTION
**Pull request by m-wynn**
These packages enable Korean keyboard input via ibus: a feature requested in our void lab.

### CODE REVIEW

File: srcpkgs/ibus-hangul/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+version=1.5.0
+revision=1
+build_style=gnu-configure
+configure_args="--libexec=/usr/lib/ibus $(vopt_enable gir introspection)"
+short_desc="Korean input engine for IBus"
+hostmakedepends="pkg-config libhangul intltool automake gettext-devel libtool swig $(vopt_if gir gobject-introspection)"
+makedepends="python-gobject-devel libhangul ibus-devel"
+depends="ibus libhangul gtk-update-icon-cache python-gobject"
+maintainer="Matthew Wynn <matthew@matthewwynn.com>"
+homepage="https://github.com/choehwanjin/${pkgname}"
```
Comment (Vaelatern) [inline srcpkgs/ibus-hangul/template]: Please don't use variable substitution here

File: srcpkgs/ibus-hangul/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+license="GPL-2"
+distfiles="https://github.com/choehwanjin/${pkgname}/releases/download/${version}/${pkgname}-${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/ibus-hangul/template]: Please use `${homepage}` here

**Comment by m-wynn** [comment]
Fixed, and also restricted to i686 and x86_64 to avoid the "unrecognized .so format" thrown by travis.

I'm not sure why it's failing on non-musl.  Could it be because of the bootstrapping segfault on the locale generation?  It builds fine on my glibc machine, last I checked.

File: srcpkgs/ibus-hangul/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+short_desc="Korean input engine for IBus"
+hostmakedepends="pkg-config libhangul intltool automake gettext-devel libtool swig $(vopt_if gir gobject-introspection)"
+makedepends="python-gobject-devel libhangul ibus-devel"
+depends="ibus libhangul gtk-update-icon-cache python-gobject"
+maintainer="Matthew Wynn <matthew@matthewwynn.com>"
+homepage="https://github.com/choehwanjin/ibus-hangul"
+license="GPL-2"
+distfiles="${homepage}/releases/download/${version}/${pkgname}-${version}.tar.gz"
+checksum=a22d4030bb20f73914a9d8a28c71b823123a5d1902a30f8328111af50f4f1788
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/ibus-hangul/template]: Please do not have this line

File: srcpkgs/libhangul/template [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+version=0.1.0
+revision=1
+build_style=gnu-configure
+short_desc="A library to support hangul input and character classification"
+maintainer="Matthew Wynn <matthew@matthewwynn.com>"
+homepage="http://code.google.com/p/libhangul/"
+license="LGPL-2.1"
+distfiles="https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum=5905aa5c557d8d46e190fec1afdd75babf3bc086ef286b42007572703658295a
+only_for_archs="i686 x86_64"
```
Comment (Vaelatern) [inline srcpkgs/libhangul/template]: Please do not restrict in this way unless downloading binaries for specific arches.

**Comment by Vaelatern** [comment]
Travis is choking. I'll give it a shot on my system later and try to fix the cross building then too.

**Comment by m-wynn** [comment]
Alright.  Having more trouble with armv7 and friends, which is why I initially restricted it to x86_64 and i686.  Let me know what you find.

**Comment by Vaelatern** [comment]
Those are cross build errors, likely related to calling plain old 'gcc' instead of our $CC compiler

**Comment by m-wynn** [comment]
Any idea why this is failing?  I'm unable to view the travis logs.

**Comment by m-wynn** [comment]
Got it building!

**Pull request closed without merging**